.github/dependabot.yml

```
@@ -117,6 +117,24 @@ updates:
     schedule:
       interval: "weekly"
       day: "sunday"
+  -
+    package-ecosystem: "gomod"
+    directory: "/instrumentation/github.com/99designs/gqlgen/otelgqlgen"
+    labels:
+      - dependencies
+      - go
+      - "Skip Changelog"
+    schedule:
+      interval: "weekly"
+      day: "sunday"
+  - package-ecosystem: "gomod"
```

please make the formatting consistent

```suggestion
  - 
    package-ecosystem: "gomod"
```